src/test/java/com/katelocate/menugenerator/RecipeControllerTests.java

```diff
@@ -68,8 +82,64 @@ static void configureProperties(DynamicPropertyRegistry registry) {
 
     @BeforeEach
     void setUp() {
-        RestAssured.baseURI = "http://localhost:" + port;
+        RestAssured.baseURI = "http://localhost:" + port + "/api/recipes";
         recipeRepository.deleteAll();
     }
 
+    List<Recipe> testRecipes = List.of(
+            new Recipe(0, "Breakfast", RecipeType.BREAKFAST, "Scramble eggs"),
+            new Recipe(1, "Dinner", RecipeType.DINNER, "Burrito"),
+            new Recipe(2, "Snack", RecipeType.SNACK, "Potato chips"),
+            new Recipe(3, "Dessert", RecipeType.DESSERT, "Mochi")
+    );
+
+    static final Logger logger = Logger.getLogger(RecipeControllerTests.class.getName());
+    @Test
+    void shouldFindAll() {
+        recipeRepository.saveAll(testRecipes);
+
+        given()
+                .contentType(ContentType.JSON)
+                .when()
+                    .get("")
+                .then()
+                    .statusCode(200)
+                    .body(".", hasSize(4));
```

Comment:
nit: I think it can be something like `hasSize(testRecipes.size())` or what is the correct Java syntax